src/certificate-roots.toit

```diff
@@ -14,12 +12,52 @@ import tls
 import .get-root
 export get-root-from-exception
 
+/**
+A library of root certificates extracted from Mozilla's NSS.
+
+The most common use-case is to install the common trusted roots
+  using the $install-common-trusted-roots method. On devices that have
+  more memory, you can also install all known trusted roots using
+  the $install-all-trusted-roots method.
+
+If you want to minimize the size of the program and memory, you can also
+  selectively install individual roots by calling 'install' on them.
+
+Some APIs also accept a list of root certificates, but installing roots
+  globally is typically more convenient.
```

Comment:
It really isn't globally? It is per process, right?

Reply:
done.